### ISSUE TITLE
Conversion 2D to 3D from 1260p minimum in the dev branch

### ISSUE THREAD

**Issue by John9520** [nagadomi/nunif#525]
Hello,

Since I use the dev branch in Iw3 Desktop, I need to have a minimum of 1260p for the 2D to 3D conversion to be functional, I tested the master branch and the problem is not here. Something is interfering with the conversion in itself. Can you look into it?
I use Video Depth Anything Stream Metric in the dev branch and Video Depth Anything Stream in the master branch, I have already tested Video Depth Anything Stream in the dev branch and the problem is always here.

Thank you

**Comment by nagadomi** [comment]
What is the specific resolution and error message?
The video I often use for testing is 720x720, and it works without any problems.

**Comment by nagadomi** [comment]
If the error message is "thread is already dead" you can retrieve the error message using the following steps.
 
1. Run `nunif-prompt.bat`
2. Type `python -m iw3.desktop.gui` and Enter

Error messages are displayed on the console (command prompt).

**Comment by John9520** [comment]
I don't get message it's just the conversion in himseft who doesn't do anything, no 3D.
I have reinstalled Iw3 and get the same result on the dev branch.

This is what I get on the prompt:
*\Desktop\nunif-master\windows_package\python\Lib\site-packages\torch\_inductor\lowering.py:7007: UserWarning:
Online softmax is disabled on the fly since Inductor decides to
split the reduction. Cut an issue to PyTorch if this is an
important use case and you want to speed it up with online
softmax.

  warnings.warn(

The resolution I use is 2306x1260 for be functional, 1920x1080 is not okay.
The depth resolution is 256x256.

**Comment by nagadomi** [comment]
Try disabling `torch.compile`.
If the GUI fails to launch, delete `nunif\tmp\iw3-desktop.cfg`

**Comment by John9520** [comment]
I guess the problem is not from torch.compile because a new reinstallation did that the problem is always here on the dev branch.
I has tested the inpaint branch too but it's the same as dev since it was merged.

**Comment by nagadomi** [comment]
Your console log shows a warning from torch.compile, which indicates that it is not a normal installation environment. It needs to be excluded from the start.
If it crashes silently, it is likely related to the issue in https://github.com/nagadomi/nunif/issues/494
Does the same problem occur when using `GPU JPEG` option?

**Comment by John9520** [comment]
I had done a renewal installation before writing my comment without torch.compile and the problem is still there, I had uninstalled tritton which was installed on my computer before that with the .bat of one of the thread and deleted the libs folder and Include folder on my first installation. But the problem is still here, so that’s why I assume that torch.compile is not the problem because I done a renewal installation without nothing.
When using GPU JPEG option or without the result is the same on 1080p but 1260p is functional.
I get no crash.

**Comment by nagadomi** [comment]
I made one change. Please check if anything changes.

I don't have a Windows machine with a decent GPU, so I basically can't test Windows behavior.
I can only test device=CPU on a virtual machine.

**Comment by John9520** [comment]
I just tested, this is okay now.
Thank you nagadomi.

**Comment by nagadomi** [comment]
Thank you for the test.
I made one more change. If there are no issues, this should resolve it.
